pkg/controller/volume/persistentvolume/controller.go

```diff
@@ -1172,7 +1181,7 @@ func (ctrl *PersistentVolumeController) doDeleteVolume(volume *api.PersistentVol
 
 	if err = deleter.Delete(); err != nil {
 		// Deleter failed
-		return fmt.Errorf("Delete of volume %q failed: %v", volume.Name, err)
+		return err
```

Comment:
Because it would replace `deletedVolumeInUseError` with a generic one. The error message is sent to the PV as event (=volume name does not need to be logged there) and in logs there is "deletion of volume %q failed: %v"" at https://github.com/jsafrane/kubernetes/blob/master/pkg/controller/volume/persistentvolume/controller.go#L1069